src/stats.ts

```diff
@@ -92,35 +92,11 @@ export function createOnCacheHitIncrementCallsAndHits(options: Options) {
 export function getDefaultProfileName(
     fn: Fn | FunctionalComponent<Record<string, unknown>>
 ) {
-    const stack = new Error().stack;
```

Comment:
The use of `stack` was always meant to provide some additional context as to the original function, however the additional context is not worth the potential memory consumption cost created.